### ISSUE TITLE
Is this repo looking for a maintainer?

### ISSUE THREAD

**Issue by mailtruck** [hpcloud/tail#150]
Hey this project needs some basic maintenance (releasing a tag to allow dependency managers to properly install via a fix that has been merged into the master branch)

I'm happy to do it if you add me as a maintainer to the repo ;)

**Comment by mailtruck** [comment]
https://github.com/hpcloud/tail/issues/146

**Comment by wijowa** [comment]
Agreed, last tag April 2016 with many improvements merged since then.

**Comment by m3nu** [comment]
The organization `hpcloud` has 0 members at the moment. Maybe time to decide an official fork. Or is there a process to add new contributors to abandoned repos?

**Comment by nxadm** [comment]
I like and use this lib. Because the project looks dormant I started merging PRs in [my forked repo](https://github.com/nxadm/tail). I also updated the build up to the latest go release (1.13) and made the repo available for go mod. Work in progress, but I put already a release out with 4 PRs applied.

**Comment by m3nu** [comment]
Great. Will switch over. Really sad that none of the old maintainers is available to make this an orderly process.

**Comment by Pandry** [comment]
> I like and use this lib. Because the project looks dormant I started merging PRs in [my forked repo](https://github.com/nxadm/tail). I also updated the build up to the latest go release (1.13) and made the repo available for go mod. Work in progress, but I put already a release out with 4 PRs applied.

Damn, I did the same a while ago!

**Comment by nxadm** [comment]
@Pandry I merged a lot of PRs (evaluating them [here](https://github.com/nxadm/tail/issues/5) and started looking at issues [here](https://github.com/nxadm/tail/issues/6)). I am certainly open to pooling resources. For this reason, I reserved the org go-tail for _in case_ my repo would get any traction and more people would like to collaborate.

**Comment by Pandry** [comment]
@nxadm Actually, I did the same
I will look to see the differences between out versions, and in case look forward to collaborate, if you want to

**Comment by nxadm** [comment]
@Pandry Looking forward to it. Feel free to ask about the changes we made (or didn't: PR not merged).
